### DISCUSSION TITLE
Scalar timeseries dimension

### DISCUSSION THREAD

**Discussion by Beforerr** [SciQLop/speasy#156] [Q&A] [answered]
* Speasy version: 1.4

### Description

The scalar timeseries data should only have one dimension. But ndim is 2. I am not sure if this is a bug or feature.

### What I Did

```
products = "cda/THB_L2_MOM/thb_peim_densityQ"
timerage = ["2016-05-20T17:47:00", "2016-05-20T17:53:40"]
d = spz.get_data(products, timerage)
d.ndim == 2
```

**Comment by jeandet** [comment]
@Beforerr, yes this was a design choice I made at the beginning to be consistent with what Pandas does.

**Comment by Beforerr** [comment]
Thanks for explanation. Then is there any reliable way to tell one time
series is of scalar type?

**Comment by jeandet** [comment, marked as answer]
I would say:

```Python
def is_scalar(v):
    return len(v.shape)==2 and v.shape[1]==1
```
